public/template.html

```diff
@@ -109,54 +188,51 @@
                     <th scope="col" class="p-1" style="text-align: right;padding-right: 4px">Amount</th>
                 </tr>
             </thead>
             <tbody>
-                <tr style="font-family:'Montserrat', 'sans-serif';font-size:10px">
+                <tr id="tableBodyRow">
                     <td class="p-1">Basic Pay</td>
-
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">{{this.basepay}}</td>
+                    <td id="tableBodyData">{{this.basicPay}}</td>
                     <td class="p-1">TDS</td>
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">20020</td>
+                    <td id="tableBodyNextData">{{this.newTDS}}</td>
                 </tr>
-                <tr style="font-family:'Montserrat', 'sans-serif';font-size:10px">
+                <tr id="tableBodyRow">
                     <td class="p-1">Dearness Allowances</td>
-
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">{{this.DA}}</td>
+                    <td id="tableBodyData">{{this.DA}}</td>
                     <td class="p-1">Provident Fund</td>
-                    <td style="text-align: right;font-weight: bold;padding-right: 4px">{{this.PF}}</td>
+                    <td id="tableBodyNextData">{{this.PF}}</td>
                 </tr>
-                <tr style="font-family:'Montserrat', 'sans-serif';font-size:10px">
+                <tr id="tableBodyRow">
                     <td class="p-1">House Rent Allowances</td>
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">{{this.HRA}}</td>
+                    <td id="tableBodyData">{{this.HRA}}</td>
                     <td class="p-1">Professional Tax</td>
-                    <td style="text-align: right;font-weight: bold;padding-right: 4px">{{this.ProfessionalTax}}</td>
+                    <td id="tableBodyNextData">{{this.ProfessionalTax}}</td>
                 </tr>
-                <tr style="font-family:'Montserrat', 'sans-serif';font-size:10px">
+                <tr id="tableBodyRow">
                     <td class="p-1">Special Allowances</td>
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">{{this.specialAllowance}}</td>
+                    <td id="tableBodyData">{{this.specialAllowance}}</td>
                 </tr>
-                <tr style="font-family:'Montserrat', 'sans-serif';font-size:10px;border-top: 1px solid black;font-weight: bold">
+                <tr id="tableBodyLastRow">
                     <td class="p-1">Total Earnings</td>
-                    <td style="text-align:right;border-right:1px solid black;font-weight: bold;padding-right: 4px">{{this.netPay}}</td>
+                    <td id="tableBodyData">{{this.netPay}}</td>
```

Comment:
HTML IDs should be unique. If you want something common for multiple elements use classes.